src/main/java/com/amazon/opendistroforelasticsearch/jdbc/ConnectionImpl.java

```diff
@@ -279,7 +279,7 @@ private void validateResultSetHoldability(int holdability) throws SQLException {
     public PreparedStatement prepareStatement(String sql, int resultSetType, int resultSetConcurrency) throws SQLException {
         log.debug(() -> logEntry("prepareStatement (%s, %d, %d)", sql, resultSetType, resultSetConcurrency));
         checkOpen();
-        validateResultSetCharacteristics(resultSetType, resultSetConcurrency, resultSetConcurrency);
+        validateResultSetCharacteristics(resultSetType, resultSetConcurrency, ResultSet.HOLD_CURSORS_OVER_COMMIT);
```

Comment:
Could it be changed by using getHoldability()? Ref [JavaDoc](https://docs.oracle.com/javase/7/docs/api/java/sql/Connection.html#prepareStatement(java.lang.String,%20int,%20int))
> The holdability of the created result sets can be determined by calling getHoldability().

Reply:
No. JavaDoc says client can  determine holdability by calling `ResultSet.getHoldability()` which defaults to `ResultSet.HOLD_CURSORS_OVER_COMMIT ` in our case (Ref: https://github.com/opendistro-for-elasticsearch/sql-jdbc/blob/85112d09e0bbddfd831aa1e0801d335a0f957b1f/src/main/java/com/amazon/opendistroforelasticsearch/jdbc/ResultSetImpl.java#L1127)